src/numeric/processes.jl

```diff
@@ -90,7 +90,7 @@ MvNormal(sigma::Float64) = MvNormal(reshape([sigma], 1, 1))
 
 function discretize(mvn::MvNormal)
     n = fill(5, size(mvn.mu))
-    x, w = QE.qnwnorm(n, mvn.mu, mvn.Sigma)
+    x, w = QE.qnwnorm(n, mvn.mu, diagm(vec(mvn.Sigma)))
```

Comment:
Sorry, it was a proposal how it should be, but not implemented. Or probably I haven't understood your remark.